roles/pulp_webserver/templates/pulp-vhost.conf.j2

```diff
@@ -26,6 +26,8 @@ Define pulp-api {{ pulp_api_bind }}
   IncludeOptional pulp/*.conf
 
   # static files are served through whitenoise - http://whitenoise.evans.io/en/stable/
+  # exclude all pulp v2 api calls
+  ProxyPassMatch ^/pulp/api/v2 !
```

Comment:
cool!

Reply:
Yup this is a good workaround for now. :+1: Thanks!